databricks/koalas/tests/test_series.py

```diff
@@ -2226,3 +2226,178 @@ def test_backfill(self):
             # Test `inplace=True`
             kser.backfill(inplace=True)
             self.assert_eq(expected, kser)
+
+    def test_compare(self):
+        if LooseVersion(pd.__version__) >= LooseVersion("1.1"):
+            pser1 = pd.Series(["b", "c", np.nan, "g", np.nan])
+            pser2 = pd.Series(["a", "c", np.nan, np.nan, "h"])
+            kser1 = ks.from_pandas(pser1)
+            kser2 = ks.from_pandas(pser2)
+            self.assert_eq(
+                pser1.compare(pser2).sort_index(),
+                kser1.compare(kser2).sort_index(),
+                check_exact=False,
```

Comment:
Do we need this and the followings?
Seems like it passes without it.

Reply:
Oh, yeah we don't need it.

Thanks :)